website/src/app/(home)/widgets/page.css

```
@@ -0,0 +1,183 @@
+  line-height: 1.1;
+  background-image: linear-gradient(146deg, #000, #757a7d);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+}
+.dark .headline {
+  background-image: linear-gradient(146deg, #fff, #757a7d);
+}
+.subtitle {
+  font-size: 1.3rem;
+  font-size: min(1.3rem, max(3.5vw, 1.2rem));
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)